docs/framework/unmanaged-api/metadata/imetadataimport-getinterfaceimplprops-method.md

```diff
@@ -19,7 +19,7 @@ author: "mairaw"
 ms.author: "mairaw"
```

Comment:
given how thoroughoully this was changed, it would be good to update the ms.date value to today's date on line 3